content/docs/installation/contents.lr

```diff
@@ -6,86 +6,94 @@ sort_key: 50
+Lektor is a command line utility, and requires some terminal or command line work 
+to install. If you can read and follow instructions, you will most likely have no 
+issues.
+
+## Requirements
+
+Although `lektor` would work without Imagemagick or FFmpeg, you would be missing on 
+significant parts of its capabilities. Therefore, you need to make sure you have the 
+following software installed on your computer:
+
+* Python 3.6+ is required. If you got an older version, look at
+  {Python 3 Installation & Setup Guide}(https://realpython.com/installing-python/).
+  * On Ubuntu you basically don't need anything new.
+  * On macOS you're done as well.
+  * On Windows, make sure that Python is in your `PATH`.
+* ImageMagick. 
+  * On macOS `brew install imagemagick` can get you this.
+  * On Ubuntu `sudo apt install imagemagick` will solve it.
+  * On Windows do `choco install imagemagick`, which requires [chocolatey :ext](https://chocolatey.org/),
+    or [download from here :ext](http://www.imagemagick.org)).
+* FFmpeg (mostly for video thumbnailing).
+  * On Ubuntu you get it through `sudo apt install ffmpeg.`
+  * On macOS `brew install ffmpeg` will do the trick.
+  * On Windows you'll have to put a little [work](https://www.wikihow.com/Install-FFmpeg-on-Windows) in.
+
+## Install pipx
 
 Once you have those installed and have made sure that they are on your `PATH`, you can
-get Lektor installed with our installation script:
+get Lektor installed with `pipx`. Selcet the `pipx` install method of your preference:
 
+__On Ubuntu through `apt`.__
 ```
-# curl -sf https://www.getlektor.com/installer.py | python3
+apt update
+sudo apt install python3-venv pipx
+pipx ensurepath
 ```
-
-This will attempt to install lektor in your user's `HOME`. If you want a system-wide installation, try this instead:
-
+__On Ubuntu through `pip`.__
+```
+sudo apt install python3-pip python3-venv
+python3 -m pip install --user pipx
+python3 -m pipx ensurepath
+```
+__On macOS__
 ```
-$ curl -sf https://www.getlektor.com/installer.py | sudo python3
+brew install pipx
+pipx ensurepath
+```
+__On Windows__
+```
+python -m pip install --user pipx
+python -m pipx ensurepath
 ```
 
-If you would like to install Lektor without being prompted, set LEKTOR_SILENT before running the prior command.
+For more information on `pipx` installation go to [pypa.github.io/pipx/installation/](https://pypa.github.io/pipx/installation/)
 
-For Windows, make sure that Python is in your `PATH` and run in `Powershell`:
+### Finally Install Lektor
+
+Once installed, close terminal (or `Powershell` or  `command prompt`) for those 
+changes to take effect. Then proceed to reopen and install Lektor with pipx:
 
 ```
-PS C:\> (new-object net.webclient).DownloadString('https://www.getlektor.com/installer.py') | python
+pipx install lektor
 ```
 
-or you can use the `command prompt` instead:
+This will install lektor in your user's `HOME`. If you want a system-wide installation, try this instead:
```

Comment:
I'd remove the following steps for a system-wide installation for the reason already outlined below. This is so far from anything I'd recommend.

Reply:
I tend to agree.

But I suspect that the reason this section was included is to maintain feature parity with the current install script.

The current install script [does appear to explicitly support][1] site-wide installation.  In fact, it does so automatically, if it is run by root.  (Eek!)

[1]: <https://github.com/lektor/lektor-website/blob/236d0daee7ca7105935ebf5a0611b2709c2520c2/assets/installer.py#L37>

Reply:
Yep, it was done to keep it in line with current options. Otherwise it can be removed, it seems there are 3 votes now.